templates/trustification/includes/_configure.tpl

```diff
@@ -0,0 +1,86 @@
+{{ define "rhtap.trustification.configure" }}
+- name: configure-trustification
+  image: quay.io/codeready-toolchain/oc-client-base:latest
+  command:
+    - /bin/bash
+    - -c
+    - |
+      set -o errexit
+      set -o nounset
+      set -o pipefail
+      set -x
+
+      cd /tmp
+
+      # Installing Helm...
+      curl -fsSL https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash
+      
+      # Storing the attributes ".trustification" from "values.yaml" as a
+      # standalone file, employed later on as input for the trustification Charts.
+      declare -r TRUSTIFICATION_VALUES="/tmp/trustification-values.yaml"
+  {{ if (index .Values "trustification") }}
+      cat <<EOF >${TRUSTIFICATION_VALUES}
+      ---
+{{ index .Values "trustification" | toYaml | indent 6 }}
+      EOF
+  {{ end }}
+
+      # Same namespace where the "rhtap" is being released
+      declare -r NAMESPACE="{{ .Release.Namespace }}"
+      # Primary openshift domain name, other apps will be exposed via wildcards
+      declare -r INGRESS_DOMAIN=$(
+          oc --namespace=openshift-ingress-operator \
+            get ingresscontrollers.operator.openshift.io default \
+              --output=jsonpath='{.status.domain}'
+      )
+      # suffix for applications with a fully qualified domain
+      declare -r APP_DOMAIN="-${NAMESPACE}.${INGRESS_DOMAIN}"
+
+      # Cloning the trustification repository, and resetting to a known commit
+      # before rollout.
+      git clone https://github.com/trustification/trustification.git
+      pushd trustification &&
```

Comment:
You have `set -o errexit`, so you do not need to chain commands with &&.

Reply:
Indeed, the `errexit` flag stops the script upon command execution error. However, the idea of using the `&&` is to create a logical block showing the dependency of each command, as well the execution order.